Source/Other/CategoryAdditions/SPStringAdditions.m

```diff
@@ -43,9 +43,12 @@ + (NSString *)stringForByteSize:(long long)byteSize
 {
 	double size = byteSize;
 	
-	NSNumberFormatter *numberFormatter = [[[NSNumberFormatter alloc] init] autorelease];
-	
-	[numberFormatter setNumberStyle:NSNumberFormatterDecimalStyle];
+	static NSNumberFormatter *numberFormatter = nil;
+	static dispatch_once_t onceToken;
+	dispatch_once(&onceToken, ^{
+		numberFormatter = [[NSNumberFormatter alloc] init];
+		[numberFormatter setNumberStyle:NSNumberFormatterDecimalStyle];
```

Comment:
decimal style is used on couple of places as well

Reply:
Can you give an example please?

Reply:
in this PR you changes numerous places where number formatter has this style == is the same, so one static var in swift extension could replace them all :)

Reply:
will try the swift thing today